examples/tf-netapp-cvs/README.md

```diff
@@ -0,0 +1,74 @@
+# NetApp Cloud Volumes Service (CVS) Terraform Example
+This example shows how to deploy NetApp CVS volumes using terraform (i.e. without using any external modules).
+This code also enables replication between volumes if *replication* variable is set as *true*.
+
+## Prerequisites
+1. You must have a recent version of Terraform installed (1.1.6+).
+1. A working GCP project with billing enabled and the below APIs enabled:
+    1. NetApp Cloud Volumes API
+    1. Service Management API
+    1. Service Networking API
+    1. Service Usage API
+    1. Compute Engine API
+1. Make sure you have followed this Quickstart before using this Terraform example <https://cloud.google.com/architecture/partners/netapp-cloud-volumes/quickstart>. This will make sure that all the below prerequisites are met.
+    1. Enable Peering between NetApp and your GCP project. The steps to do this are mentioned in below section.
+
+## Enable Network Peering
+Run the below commands to establish Network Peering between your project and NetApp Project. You will also get a prompt to run these commands manually when you try to create a volume from the Qwikstart.
+![Alt text](/Enable-Network-Peering-popup.png)
+```
+export PROJECT_ID=<PROJECT_ID>
+```
+```
+export NETWORK=<VPC_NETWORK_NAME>
+```
+```
+gcloud --project=$PROJECT_ID compute addresses create \
```

Comment:
You should also use Terraform to take care of prerequisites instead of using gcloud.

Reply:
This example assumes that you already have a networking setup in place and it only demonstrates the various operations that can be performed for NetApp using Terraform like provisioning of volumes, enabling backups based on a schedule, enabling volume replication for DR scenarios etc..
The gcloud commands are to setup Private Service Access between the users VPC and NetApp VPC. This is just a one time setup that the user has to perform. https://cloud.google.com/architecture/partners/netapp-cloud-volumes/setting-up-private-services-access#setting_up_private_service_access. I have provided the links and added images in README to make it easier for the user if they haven't already set it up.